pkg/cluster/cluster_config.go

```diff
@@ -257,5 +265,54 @@ func getClusterInfo(ctx context.Context, cli client.Client) (ClusterInfo, error)
 	}
 	c.Version = ocpVersion
 
+	// Check for FIPs
+	if fipsEnabled, err := isFipsEnabled(ctx, cli); err == nil {
+		c.FipsEnabled = fipsEnabled
+	} else {
+		logf.FromContext(ctx).Info("could not determine FIPS status, defaulting to false", "error", err)
+	}
+
 	return c, nil
 }
+
+func isFipsEnabled(ctx context.Context, cli client.Client) (bool, error) {
+	// Check the install-config for the fips flag and it's value
+	// https://access.redhat.com/solutions/6525331
+	cm := &corev1.ConfigMap{}
+	namespacedName := types.NamespacedName{
+		Name:      "cluster-config-v1",
+		Namespace: "kube-system",
+	}
+
+	var err error
+
+	if err := cli.Get(ctx, namespacedName, cm); err != nil {
+		return false, err
+	}
+
+	if installConfigStr, ok := cm.Data["install-config"]; ok {
+
+		if installConfigStr == "" {
+			// No install-config found or empty
+			return false, nil
+		}
+
+		var installConfig InstallConfig
+
+		if err := yaml.Unmarshal([]byte(installConfigStr), &installConfig); err == nil {
+			return installConfig.FIPS, nil
+		}
+
+		// If unmarshaling fails, fall back to the string search
+		if strings.Contains(strings.ToLower(installConfigStr), "fips: true") {
+			return true, nil
+		}
+		if strings.Contains(strings.ToLower(installConfigStr), "fips: false") {
+			return false, nil
+		}
+		return false, fmt.Errorf("failed to unmarshal install-config: %w, falling back to string search", err)
+	}
+
+	// default to false with no error
+	return false, nil
```

Comment:
it can probably simplified to something like:

```go
installConfigStr := cm.Data["install-config"]
if installConfigStr == "" {
	// No install-config found or empty
	return false, nil
}

var installConfig InstallConfig
if err := yaml.Unmarshal([]byte(installConfigStr), &installConfig); err != nil {
        // If unmarshaling fails, fall back to the string search
        // swallow the error
	return strings.Contains(strings.ToLower(installConfigStr), "fips: true"), nil
}

return installConfig.FIPS, nil
```